app/models/user.rb

```diff
@@ -56,7 +56,7 @@ class User < ApplicationRecord
       attr_accessor :_skip_creating_profile
 
       # All new users should automatically have a profile
-      after_create_commit -> { Profile.create(user: self) }, unless: :_skip_creating_profile
+      after_create_commit :create_user_profile, unless: :_skip_creating_profile
```

Comment:
Is there a specific reason for turning this lambda into a method? I wanted to have all temporary profile-related code within this inline concern but apart from that, they are functionally equivalent.

Reply:
I would revert this change as well, given that it's part of the included concern and not strictly part of the `User` model